docs/howto/custom-management-commands.txt

```diff
@@ -208,12 +207,10 @@ Attributes
 All attributes can be set in your derived class and can be used in
 :class:`BaseCommand`’s :ref:`subclasses<ref-basecommand-subclasses>`.
 
-.. attribute:: BaseCommand.can_import_settings
+.. versionchanged:: 1.11
 
-    A boolean indicating whether the command needs to be able to
-    import Django settings; if ``True``, ``execute()`` will verify
-    that this is possible before proceeding. Default value is
-    ``True``.
+    The ``can_import_settings`` attribute has been removed as it was not used
```

Comment:
Put this in the release notes instead. "The unused BaseCommand.can_import_settings attribute is removed."